api/GPUDevice.json

```diff
@@ -357,6 +357,46 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "optional_entryPoint": {
+          "__compat": {
+            "description": "<code>entryPoint</code> properties are optional for determined default compute shader entry points.",
```

Comment:
@chrisdavidmills Is this behavioral sub-feature referring to [this part](https://gpuweb.github.io/gpuweb/#:~:text=Since%20the%20entryPoint%20dictionary%20member%20is%20not%20required) of the spec?

> Since the [entryPoint](https://gpuweb.github.io/gpuweb/#dom-gpuprogrammablestage-entrypoint) dictionary member is not required, methods which consume a [GPUProgrammableStage](https://gpuweb.github.io/gpuweb/#gpuprogrammablestage) must use the "[get the entry point](https://gpuweb.github.io/gpuweb/#abstract-opdef-get-the-entry-point)" algorithm to determine which entry point it refers to.

Unless the spec has changed: If Chromium previously required an `entryPoint`, I would rather expect that we add a support entry on the parent feature with a note.

Reply:
The spec did change; see https://github.com/gpuweb/gpuweb/pull/4387